src/components/CreatePollForm.vue

```diff
@@ -0,0 +1,304 @@
+        </div>
+      </div>
+      <small v-if="v$.pollData.polledHouses.$error" id="poll_polledHouses" class="p-error">{{
+        v$.pollData.polledHouses.$errors[0].$message
+      }}</small>
+    </div>
+    <div class="input-section">
+      <label class="dialog-item" for="caledar-begin">Дата початку:</label>
+      <Calendar
+        id="caledar-begin"
+        dateFormat="dd.mm.yy"
+        v-model="startDate"
+        :showIcon="true"
+        :minDate="minDate"
+        @date-select="ChangeDate"
+      />
+    </div>
+    <div class="input-section">
+      <label class="dialog-item" for="calendar-finish">Дата закінчення:</label>
+      <InputText id="calendar-finish" disabled="true" :value="pollData.completionDate"></InputText>
+      <small id="apartment_area_help" class="p-warning">виставляється автоматично 15 днів з дати початку</small>
+    </div>
+    <div class="button-div">
+      <Button
+        id="save-button"
+        label="Додати опитування"
+        icon="pi pi-check"
+        autofocus
+        class="p-button-info"
+        type="submit"
+        value="Submit"
+      />
+      <Button
+        id="cancel-button"
+        label="Відмінити"
+        icon="pi pi-times"
+        class="p-button-outlined p-button-info"
+        @click="cancelEditing"
+      />
+    </div>
+  </form>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+import useVuelidate from '@vuelidate/core';
+import {
+  requiredValidator,
+  pollTitleLenghtValidator,
+  pollDescriptionLenghtValidator,
+  cyrillicLangTitleValidator,
+} from '@/utils/validators';
+import Button from 'primevue/button';
+import Textarea from 'primevue/textarea';
+import Checkbox from 'primevue/checkbox';
+import Calendar from 'primevue/calendar';
+import { StoreModuleEnum } from '@/store/types';
+import { HousesActionsEnum, HousesGettersEnum } from '@/store/houses/types';
+import { HouseModel } from '@/shared/models/house.model';
+import { PollsActionEnum } from '@/store/polls/types';
+import InputText from 'primevue/inputtext';
+
+export default defineComponent({
+  name: 'CreatePollForm',
+  components: {
+    Button,
+    Textarea,
+    Checkbox,
+    Calendar,
+    InputText,
+  },
+  props: {
+    cooperationId: {
+      type: Number,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      pollData: {
+        title: '',
+        description: '',
+        polledHouses: [],
+        creationDate: new Date(),
+        completionDate: '',
+      },
+      startDate: null,
+      finishDate: new Date(),
+      displayCreatePollModal: false,
+      minDate: new Date(),
+      submitted: false,
+      v$: useVuelidate(),
+    };
+  },
+  validations() {
+    return {
+      pollData: {
+        title: { requiredValidator, pollTitleLenghtValidator, cyrillicLangTitleValidator },
+        description: { requiredValidator, pollDescriptionLenghtValidator, cyrillicLangTitleValidator },
+        polledHouses: { requiredValidator },
+      },
```

Comment:
code duplication in #311 PR
consider moving it to validation utils, to be reusable
also specify a common name in line 142